README.md

```diff
@@ -32,7 +32,7 @@
 
 **Updates:**
 
-**June 30 2024, embodied-agents v1.0.0**:
+**June 30 2024, embodied-agents v1.0**:
```

Comment:
increment to match pypi?

Reply:
Nah, we may want to have minor updates/fixes like v1.0... This should just be the general version.